content/docs/components/jupyter.md

```diff
@@ -99,6 +99,19 @@ This means that you can directly create these k8s resources directly from your j
 ```
 !kubectl create -f myspec.yaml
 ```
+## Creating a custom Jupyter image
+You can create your own Jupyter image and use it in your Kubeflow cluster.
+Your custom image needs to meet the requirements created by Kubeflow Notebook Controller. Kubeflow Notebook Controller  manages the life-cycle of notebooks.
+ Kubeflow Web UI expects the Jupyer to be launched upon spawning a Jupyter notebook. For that you need to set the default command of your image to launch Jupyter. Tha Jupyter launch command needs to be set as follows:
```

Comment:
"Kubeflow Web UI expects the Jupyer to be launched upon spawning a Jupyter notebook" seems weird to me. Suggest "... to be launched upon running the docker image with only `docker run`" to make it clear